public/views/index.html

```diff
@@ -281,20 +282,70 @@ <h5 id="contactModalTitle" class="modal-title">
             <div class="form-group">
               <label for="control-label">Email</label>
               <input type="text" class="form-control" name="email" ng-model="email" placeholder="email address">
+                <div ng-show="hasError" class="text-danger">
+                  {{hasError.email}}
+                </div>
+            </div>
+            <div class="form-group">
+              <label for="control-label">Password</label>
+              <input type="password" class="form-control" name="password" ng-model="password" placeholder="password">
               <div ng-show="hasError" class="text-danger">
-                {{hasError.email}}
+                {{hasError.password}}
               </div>
             </div>
+            <hr>
+            <button class="btn btn-login btn-lg btn-block" type="submit">
+              Sign In
+            </button>
+            <span ng-click="toggleResetModal()" class="pointer nav-link text-center forgot-password" data-toggle="modal" data-target="#resetModal">forgot password?</span>
+          </form>
```

Comment:
Hi Samson, good work on the password reset feature. But it seems a non-authenticated user can change the password of an authenticated user. Why not send a password reset link via email to users who forget their password. That way, you are sure only the owner of an account can reset its password.